### ISSUE TITLE
Global Series: Jiang Yanggu & Mu Yanling will introduce a second standard environment

### ISSUE THREAD

**Issue by NilsEnevoldsen** [glacials/whatsinstandard#79]
This isn't an issue yet, I'm just bringing this to your attention so you have time to think about it and people can offer comments if desired. The definition of Standard is going to become geographically heterogeneous.

> As such, we're trying something a little different with this product. While the cards in it are not designed for serious tournament play, we wanted to make sure the fans it was created for—new, Chinese-speaking fans—could still play with the cards at an Friday Night Magic when they're starting out. So, with this in mind, **_Global Series: Jiang Yanggu & Mu Yanling_ will only be Standard legal in China**. It will not be Standard legal anywhere else. Like every other product we release, the cards will be legal for Legacy and Vintage use. They will not be legal in Modern.

https://magic.wizards.com/en/articles/archive/news/global-series-meet-jiang-yanggu-and-mu-yanling-2018-05-18

**Comment by glacials** [comment]
uhhhhhh

**Comment by NilsEnevoldsen** [comment]
Hot take: I'd guess well over 99% of site visitors are not interested in Chinese Standard. Have a "show additional Chinese Standard sets" disclosure link, similar to "show recently dropped sets".

**Comment by glacials** [comment]
Yeah thanks for the heads up, that's a good suggestion. Although I will add this to #39 so that if/when we localize into Chinese we can show it differently or something. This feels like a really weird decision from WotC and they don't do a great job explaining the rationale for not making it legal globally.
